form/root_storage/root_tfile.cpp

```diff
@@ -15,8 +15,4 @@ ROOT_TFileImp::ROOT_TFileImp(const std::string& name, char mode) :
   }
 }
 
-//Trivial destructor is sufficient because all members managed by the stack.
-//Writing it here lets us move the dependence on the TFile implementation out of the header.
-ROOT_TFileImp::~ROOT_TFileImp() {}
```

Comment:
Why move this back in the header file?  If you want to use the default-generated destructor, you can do that here in the `.cpp` file:

```cpp
ROOT_TFileImp::~ROOT_TFileImp() = default;
```